GitUI/FilterBranchHelper.cs

```diff
@@ -8,6 +8,7 @@ namespace GitUI
 {
     public class FilterBranchHelper : IDisposable
     {
+        private bool _applyingFilter;
```

Comment:
Init?

Reply:
fields are automatically initialised; ref types to `null`, value types to their default value (in this case it is `false`m which is we want)